src/main/resources/data/minecolonies/tags/items/blacksmith_product.json

```diff
@@ -2,6 +2,7 @@
   "replace" : false,
   "values" : [
     "#forge:nuggets",
-    "#forge:ingots"
+    "#forge:ingots",
+    "minecraft:shears"
```

Comment:
Why put it here, rather than putting #forge_ingots in blacksmith_ingredient.json so that it picks up shears, flint and steel, and anything added by mods? I'm assuming ingots are in products for the purposes of compacting nuggets to ingots. But if you moved all of this to ingredient, you'd still get that. That change would open up lamps here, so you'd want to exclude torches from the blacksmith.

Reply:
Because then you could craft anything with ingots like armor for example which is what the blacksmith is for or pistons instead of mechanic.

Reply:
If you add ingots to the blacksmith ingredients, and redstone and torches to the blacksmith exclude, it seems like you'd get the exact set of things that you'd expect from the blacksmith?

Reply:
still have to deal with lanterns and rails and hoppers and minecarts and. tripwire

Reply:
Lanterns are covered by the torches ingredient exclusion, rails are going to have to be a product exclusion, as the ingredients are the same as an iron sword.  For the tripwire, a product exclusion might be the easiest solution, as a planks exclusion would conflict with the wooden tools. 

If you can exclude mechanic_product from the blacksmith, then you easily solve moving things to the mechanic by adding them as a product there.